serving/docker/Dockerfile

```diff
@@ -11,7 +11,7 @@
 # the specific language governing permissions and limitations under the License.
 FROM ubuntu:22.04 AS base
 ARG djl_version
```

Comment:
for all - why don't we have a default for djl_version?

Reply:
djl_version bumping is centralized in the TOML file now

Reply:
yeah, we would make the change in the toml file's version for djl https://github.com/deepjavalibrary/djl-serving/blob/master/gradle/libs.versions.toml#L5. It will be different then it had been, but @brndysgit  is going to work on the documentation for the change.